### DISCUSSION TITLE
我必须要说一件事情

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage#45] [公告]
简单主页是**第一个投稿龙猫主页预设**的，所以**被龙猫第一个添加到了PCL2中**。结果，**龙猫接下来的操作**是：**简单主页作为所谓“大杂烩”种类，已经存在了第一个，后面的就没必要再加了**

鱼逝，就开始有人说简单主页搞垄断，点名表扬 @Ext1nguisher

![image](https://github.com/user-attachments/assets/305d08da-9e11-434c-93e9-cce3857c4472)

在此我想说：真特么不是我想搞的，我也跟龙猫说了 ~他不听啊我能有什么办法~

![image](https://github.com/user-attachments/assets/aad8010f-2b89-4561-adad-51f07861d845)

![image](https://github.com/user-attachments/assets/06a00476-98ca-487c-9d7d-eb054e850f99)

而且为了让各位的主页能够像加入预设的那样，我也开了另一个项目，主页市场

![image](https://github.com/user-attachments/assets/877d6fe5-adb2-464b-97c2-efcfa78978cd)

但是，我的主页的风评依旧不太好，~还是天天打喷嚏~

所以别骂了别骂了，要骂我先说通龙猫吧，我也是学生党不容易啊自己抽空做了个简单主页我也没想到就变成唯一的简单主页了啊啊啊啊啊啊啊啊啊啊啊啊

**Comment by WForst-Breeze** [comment]
惨惨的 TwT 支持一下，主页做的很棒，可能就是一些人不太了解情况，跟他们解释清楚肯定就没问题了！某种意义上这也促进了预设主页们的良性竞争，可以让主页的质量越来越好，主页市场也是个挺不错的想法，加油~ 🥳

**Reply by MFn233** [reply]
谢谢理解！！QWQ

**Reply by Ad-closeNN** [reply]
既然在这就不妨说一个小小小小的优化吧（直接改就行
这玩意开头是小写的，其他都是大写的 ~~qaq看着好难受~~
![image](https://github.com/user-attachments/assets/7530917a-7829-454e-bb0b-354752bb7bda)

**Reply by wuliaodexiaoluo** [reply]
![image](https://github.com/user-attachments/assets/94709e62-4e77-484d-9791-82f78f503c13)

前来支持下，除非主页测试，现在大部分时候挂的都是简单主页
（除此之外，其他的大杂烩我都不喜欢（个人感觉不够简洁，而且也不太喜欢主页内的背景配色，还有些态度都不行的））

>跟他们解释清楚肯定就没问题了

不好评价，但总感觉太理想化了？可能只适合君子？
~（社交恐怖分子）~

**Reply by WForst-Breeze** [reply]
> >跟他们解释清楚肯定就没问题了
> 
> 不好评价，但总感觉太理想化了？可能只适合君子？
> ~（社交恐怖分子）~

清者自清，我们只要解释清楚了这件事情，能明白的自然会明白。那些解释过后依旧追着不放的，再如何费力地解释也都是徒劳，只要做好自己就没问题的 😘 我也相信来到这里的人，都是君子多而小人少的

**Reply by Neclyon** [reply]
那我骂龙猫人家肯定也不听啊，~~我总得找个挨骂的人吧~~
再说我说的那句话都配上划去线了，难道还要狗头护体??!

**Reply by MFn233** [reply]
~我好像也没说你啥吧~@Ext1nguisher

**Reply by Neclyon** [reply]
> ~我好像也没说你啥吧~@Ext1nguisher

那就谢谢你的点名表扬啊

**Reply by wuliaodexiaoluo** [reply]
>那我骂龙猫人家肯定也不听啊，我总得找个挨骂的人吧

鉴定为闲的，建议去玩舞萌DX

>再说我说的那句话都配上划去线了，难道还要狗头护体??!

~在我看来你发的帖子跟挑衅没区别~

>那就谢谢你的点名表扬啊

那就谢谢你给我邮箱刷两条邮件啊

**Reply by Neclyon** [reply]
> > 那我骂龙猫人家肯定也不听啊，我总得找个挨骂的人吧
> 
> 鉴定为闲的，建议去玩舞萌DX
> 
> > 再说我说的那句话都配上划去线了，难道还要狗头护体??!
> 
> ~在我看来你发的帖子跟挑衅没区别~
> 
> > 那就谢谢你的点名表扬啊
> 
> 那就谢谢你给我邮箱刷两条邮件啊

?啥意思?
所以我对于你这种人来说说什么都是挑衅的
你除了夸奖的什么都听不进去
疯狗乱咬

**Reply by Neclyon** [reply]
我声明一下，我整段帖子对于MFn和简单主页没有任何意见。
有些人好像就是不喜欢和谐的环境，这种人建议去玩快手。你要知道，**能上的去github是你的福气**！
我骂的是那个万恶的https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112 啊，我整篇文章重点真的就只 不 过 是 想**预言一波**，在你们看来我变成专门为了挑衅发帖了。我整篇文章只有只言片语提到了"因为万恶的https://github.com/Hex-Dragon/PCL2/discussions/2528#discussioncomment-9641112"
我说的又不是"因为万恶的简单主页"??!
我真的不知道你们是怎么看事情的，但是我发这篇帖子**绝**  **对**  **没**  **有**  **任**  **何**  **恶**  **意**  **啊**。
~~我还没有反击道德绑架的经验~~......

**Reply by MFn233** [reply]
？不知道你们在吵什么

**Reply by MFn233** [reply]
人家MFn都没说什么

**Reply by Ad-closeNN** [reply]
我从头到尾都没看懂是什么意思...

**Reply by WForst-Breeze** [reply]
怎么在我评论下面吵起来了…… 我也没有说谁就是对谁就是错，只是说解释清楚了误会就能解开了而已……

**Comment by MFn233** [comment]
@Ext1nguisher @wuliaodexiaoluo 别吵啦别吵啦，下次发帖加颜文字不就行了(￣﹃￣)

**Reply by Neclyon** [reply]
颜文字这种xxs还是能找到骂点的
总能找到骂点的，疯狗乱咬人

**Reply by MFn233** [reply]
> 颜文字这种xxs还是能找到骂点的
> 总能找到骂点的，疯狗乱咬人

也不看看回复你的是谁，你得自己找找问题了，说的话太尖锐了看不出语气😕

**Comment by SqrtTan** [comment]
咋回事？一个月没上GitHub咋这样了

**Reply by MFn233** [reply]
我的简单主页是唯一大杂烩主页，然后就引起了很多人的不满。。。

**Comment by Neclyon** [comment]
### 对不起！
对不起各位，我已经修改我的帖子，以后不再对简单主页发表任何意见。
是我的问题，我开玩笑没有顾及到大家的感受。向大家致以最诚恳的道歉。
我不是有意造成大家的困扰的，谢谢大家的理解与包容。
我的主页根本无法与简单主页相媲美，我会好好改进我的主页再来参选。
